src/object/player.cpp

```diff
@@ -1173,6 +1184,11 @@ Player::apply_friction()
     friction *= (ICE_FRICTION_MULTIPLIER*(m_sliding ? 4.f : m_stone ? 5.f : 1.f));
   else
     friction *= (NORMAL_FRICTION_MULTIPLIER*(m_sliding ? 0.8f : m_stone ? 0.4f : 1.f));
+
+  // Air friction does not make sense when the air is moving with you!
```

Comment:
Air drag depends on the object speed with respect to the air. If the velocity of the air is equal to the velocity of Tux, then indeed, there is no drag. However, when they have both different velocities, there is drag. That is actually the reason why wind blows object away.

However, it looks like the term friction stands for the friction between Tux and the ground. In such a case, the friction is one force and the air drag is another force.